### PR TITLE
Implementation bluetooth-transport based on WinAPI

### PR DESCRIPTION
**Pull request by nk0leg**
Close-Issue: [SDLWIN-383](https://adc.luxoft.com/jira/browse/SDLWIN-383)

### CODE REVIEW

**Comment by nk0leg** [comment]
@malirod, @anosach-luxoft  please review

**Comment by nk0leg** [comment]
@malirod See 2e6df94

File: src/components/transport_manager/include/transport_manager/bluetooth/bluetooth_socket_connection.h [hunk 1]
```diff
@@ -37,6 +37,7 @@
 #define SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLUETOOTH_BLUETOOTH_SOCKET_CONNECTION_H_
 
 #include "transport_manager/transport_adapter/threaded_socket_connection.h"
```
Comment (anosach-luxoft) [inline src/components/transport_manager/include/transport_manager/bluetooth/bluetooth_socket_connection.h]: @nk0leg why do we need this header?

Comment (nk0leg) [inline src/components/transport_manager/include/transport_manager/bluetooth/bluetooth_socket_connection.h]: @anosach-luxoft see 3214bce

**Pull request merged**